README.md

```diff
@@ -13,6 +13,7 @@ Inteli OpenAPI service for interacting with the DSCP (Digital Supply-Chain Platf
 | LOG_LEVEL                    |    N     |                       `info`                        | Logging level. Valid values are [`trace`, `debug`, `info`, `warn`, `error`, `fatal`] |
 | API_VERSION                  |    N     |                          -                          | API version                                                                          |
 | API_MAJOR_VERSION            |    N     |                          -                          | API major version                                                                    |
+| POLKADOT_API                 |    N     |               `http://localhost:3001`               | DSCP-API hostname for substrate node                                                 |
```

Comment:
Is this supposed to refer to the API provided by `dscp-node` or by `dscp-api` ?

Reply:
by dscp-api

Reply:
currently it's DSCP_API_HOST and DSCP_API_PORT

Reply:
ok leave it as DSCP_API_HOST and DSCP_API_PORT.

It should not be Polkadot in any case.